src/particles/PlasmaParticleContainer.cpp

```diff
@@ -116,11 +116,26 @@ PlasmaParticleContainer::ReadParameters ()
             m_u_mean[idim] = loc_array[idim];
         }
     }
-    if (queryWithParser(pp, "u_std", loc_array)) {
+    bool thermal_momentum_is_specified = queryWithParser(pp, "u_std", loc_array);
+    bool temperature_is_specified = queryWithParser(pp, "temperature_in_ev", m_temperature_in_ev);
```

Comment:
Just merging asserts.
```suggestion
    bool thermal_momentum_is_specified = queryWithParser(pp, "u_std", loc_array);
    bool temperature_is_specified = queryWithParser(pp, "temperature_in_ev", m_temperature_in_ev);
    AMREX_ALWAYS_ASSERT_WITH_MESSAGE(
        !(temperature_is_specified && thermal_momentum_is_specified),
         "Please specify exlusively either a temperature or the thermal momentum");
```